docs/hub/models-widgets.md

```diff
@@ -71,8 +71,40 @@ widget:
   example_title: Custom Speech Sample 1
 ```
 
+But even more convenient, if the file lives in the corresponding model repo, you can just use the filename or file path inside the repo:
+```yaml
+widget:
+- src: sample1.flac
+  example_title: Custom Speech Sample 1
+```
+
 We provide example inputs for some languages and most widget types in [the DefaultWidget.ts file](https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/DefaultWidget.ts). If some examples are missing, we welcome PRs from the community to add them!
 
+## Example outputs
+
+As an extension to example inputs, for each widget example, you can also optionally describe the corresponding model output, direcly in the `output` property.
+
+This is useful when the model is not yet supported by the Inference API (for instance, the model library is not yet supported, or the model is too large), so that the model page can still showcase how the model works and what results it gives.
+
+For instance, for an automatic-speec-recognition model:
+
+```yaml
+widget:
+- src: sample1.flac
+  output: "Hello my name is Julien"
+```
+
+The `output` property supports:
+- a JSON-encoded string that represents the full Inference API output
+- a YAML dictionary that represents the full Inference API output
+- (not sure) as a special case, for models that output text, a string of text
+- (not sure) as a special case, for models that output an image, a src filename to a file inside the repo.
```

Comment:
:+1: for showing an output image. When an image is returned by Inference API in json format, it's base64-encoded which is awful for a model card metadata.